src/GraphQL/Resolver/QueryType.php

```diff
@@ -204,7 +198,7 @@ public function resolveAssetGetter($value = null, $args = [], $context, ResolveI
             $this->getGraphQlService()->getLocaleService()->setLocale($args['defaultLanguage']);
         }
 
-        $assetElement = Asset::getById($args['id']);
+        $assetElement = $this->getElementByTypeAndIdOrPath($args, 'asset');
         if (!$assetElement) {
             return null;
         }
```

Comment:
@weisswurstkanone Document query (resolveDocumentGetter) does already provide fullpath additional to id. In the Document query implementation the fullpath is named 'path' though, which would make it inconsistent to our naming.
So what shall we do:
- rename 'fullpath' back to 'path' in this PR
- rename the Document query 'path' to 'fullpath' which would break backward-compatibility

Reply:
I would support both versions for now and mark the other one as deprecated (removed in 1.0). Similar to 
https://github.com/pimcore/data-hub/issues/266

Reply:
Done.
PS: well, the @deprecated annotation was not easily applicable to an 'args' argument item.

Reply:
I added a deprecation warning to the log (therefore I had to import the static Pimcore Logger to the Resolver/QueryType class though).